common/src/main/java/net/neoforged/gradle/common/extensions/DefaultJarJarFeature.java

```diff
@@ -119,6 +119,8 @@ public void createTaskAndConfiguration() {
         configuration.getAllDependencies().configureEach(dep ->
                 this.enable()
         );
+        configuration.setCanBeResolved(true);
+        configuration.setCanBeConsumed(false);
```

Comment:
In second thought, please leave a comment as to why this is needed.

Reply:
Done